### PR TITLE
Do not show sync option in audio preview.

### LINKED ISSUE #3155: [FEATURE REQUEST] Sync action in previews
There are four kind of files that are previewed inside the app:

- Images
- Text (.txt)
- Video
- Audio

Actually, the preview mode provides different actions, depending on the kind of file. We should review and unify actions for every kind of file.

This is the current status, that makes sense.

||Sync|Share|Send|Open with|Av. off|Details|Delete|
|:-:|:-:|:-:|:-:|:-:|:-:|:-:|:-:|
|Image | ❌ | ✅  | ✅  | ✅  | ✅  | ✅  | ✅  |
|Text | ❌ | ✅  | ✅  | ✅  | ✅  | ✅  | ✅  |
|Video (streaming) | ❌  | ✅  | ❌  | ❌  | ❌  | ✅  | ✅  |
|Video (downloaded) | ❌  | ✅  | ✅   | ✅   | ✅   | ✅  | ✅  |
|Audio | ✅  | ✅  | ✅  | ✅  | ✅  | ✅  | ✅ 

The main difference is that the Audio previews allow to sync the file. 

Should the sync option be removed from audio? or added in the other ones.

In my opinion, i'd remove it from audio. What do you think?

### PR DESCRIPTION
**Pull request by abelgardep**
## Related Issues
Fixes #3155 
_____

## QA

_____

## Checklist

- [x] Added changelog files for the fixed issues in folder changelog/unreleased

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,7 +21,15 @@ Details
 
    https://github.com/owncloud/android/issues/2691
    https://github.com/owncloud/android/pull/3156
-   https://github.com/owncloud/android/pull/316
+   https://github.com/owncloud/android/pull/3162
+
+* Enhancement - Align previews actions: [#3155](https://github.com/owncloud/android/issues/3155)
+
+   We've aligned the behaviour for every preview fragment. Images, videos, audios and texts
+   shows the same actions now.
```
Comment (JuancaG05) [inline CHANGELOG.md:29]: A typo here: should be "show" instead of "shows" if I'm not wrong

File: changelog/unreleased/3177 [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Enhancement: Align previews actions
+
+We've aligned the behaviour for every preview fragment. Images, videos, audios and texts shows the same actions now.
```
Comment (JuancaG05) [inline changelog/unreleased/3177:3]: Same typo as before

**Review by JuancaG05 — CHANGES_REQUESTED** [review]
Tiny change for a typo @abelgardep

**Review by JuancaG05 — APPROVED** [review]

**Comment by jesmrec** [comment]
Approved.

**Pull request merged**